serenata_toolbox/reimbursements.py

```diff
@@ -41,7 +41,7 @@ def read_csv(self, name):
     @property
     def receipts(self):
         print('Merging all datasets…')
-        datasets = ('current-year.xz', 'last-year.xz', 'previous-years.xz')
+        datasets = ["reimbursements-{}.xz".format(n) for n in range(2009, date.today().year+1)]
```

Comment:
I'm not really a python guy, so forgive me if pythonic stuff seems weird to me, I'm really new to all this. Can't we just create the YEARS constant, as suggested for CEAPDataset class and use it to reduce the line size?